coretex/cli/modules/node.py

```diff
@@ -90,14 +99,17 @@ def start(dockerImage: str, config: Dict[str, Any]) -> None:
         if initScript is not None:
             volumes.append((str(initScript), "/script/init.sh"))
 
+        limitedCpus, limitedRam, limitedSwap = docker.dockerInfo()
+        validateResources(config, limitedCpus, limitedRam, limitedSwap)
```

Comment:
This can be expanded to a `validateNodeConfig` to validate if node is properly configured, not just for resources, but for everything

Reply:
Validate the values entered for the resources during node config too, not just during node start